source/content/guides/composer-convert.md

```diff
@@ -92,18 +128,15 @@ A Composer-managed site should be able to include all custom code via Composer.
 
 When reviewing your site, take stock of exactly what versions of modules you depend on. One way to do this is to use a command like the following from within a contributed modules folder (e.g. `/modules`, etc):
 
-```bash{outputLines:2-4}
-find . -maxdepth 2 -name '*.info.yml' \
-  -exec basename '{}' '.info.yml' ';' \
-  -exec grep -e '^version' '{}' ';' \
-  -exec echo ';'
+```bash{promptUser:user}
+terminus drush $site.dev -- pm:projectinfo --fields=name,version --format=table
 ```
 
 This will list each module followed by the version of that module that is installed.
 
 You can add these modules to your new codebase using Composer by running the following for each module in the `$site-composer` directory:
 
-```{promptUser:user}
+```bash{promptUser:user}
 composer require drupal/MODULE_NAME:^VERSION
```

Comment:
It would probably be a good idea to tell folks that they should skip the ludwig module if they are using it. This module is used to integrate modules that require Composer into sites that are not composer-managed, so it might be fairly common.

Reply:
Also, note that Drush will show version strings in two different format.

If Drush says 8.x-1.8, then they should use ^1.8 (common)

If Drush says 4.0.1, then they should use ^4.0.1 (less common)

Reply:
Below, the section "#### Themes" may be removed, because it is now covered by the section above. Furthermore, the section "#### Libraries" is now largely unneeded. With Composer-managed sites, for the most part the libraries your modules will need will be included automatically by Composer. This will happen for any module that has been updated for Composer, which by now is the majority of them. There might be some modules remaining that still need external libraries, and do not use Composer to install them. These will need special handling. I'm not sure what to say in the "Libraries" section, though, since I don't have any specific examples where this is still needed.

Reply:
The section "#### Modules and Themes" should also be adjusted:

> Custom code should be checked out from the master branch, which contains the original site:
> ```
> git checkout master modules/custom
> ```
> Note that the command above does not switch branches; it copies the contents of the `modules/custom` directory from the master branch onto the current branch.
>
> Foolow suit with any other custom code you may need to carry over from other directories.